styles.css

```diff
@@ -94,7 +110,209 @@ header {
   justify-content: flex-start;
 }
 
-.icon {
+.icon,
+.icon > a {
   color: #505f79;
   margin-right: 12px;
+  font-size: 24px;
+}
+
+#works {
+  display: grid;
+  margin: 0;
+  padding: 114px 24px;
+  gap: 50px;
+  list-style-type: none;
+}
```

Comment:
- [ ] Your cards are not correctly centre 🙂, please add `justify-content: center;` to your class to fix the issue.👌